wren-base/src/main/java/io/wren/base/sqlrewrite/MetricSqlRender.java

```diff
@@ -151,21 +151,31 @@ protected String getSelectItemsExpression(Column column, Optional<String> relati
         }
 
         if (isMeasure) {
-            return format("%s AS \"%s\"", SqlFormatter.formatSql(ExpressionTreeRewriter.rewriteWith(new ExpressionRewriter<>()
-            {
-                @Override
-                public Expression rewriteIdentifier(Identifier node, Void context, ExpressionTreeRewriter<Void> treeRewriter)
-                {
-                    if (baseModel.getColumns().stream().anyMatch(c -> c.getName().equalsIgnoreCase(node.getValue()))) {
-                        return new DereferenceExpression(new Identifier(baseModel.getName(), true), new Identifier(node.getValue(), true));
-                    }
-                    return node;
-                }
-            }, Utils.parseExpression(column.getExpression()
-                    .orElseThrow(() -> new IllegalArgumentException("measure column must have expression"))))), column.getName());
+            return format("%s AS \"%s\"",
+                    awareModel(column.getExpression().orElseThrow(() -> new IllegalArgumentException("measure column must have expression")), baseModel), column.getName());
         }
 
-        return format("\"%s\".%s AS \"%s\"", relationable.getBaseObject(), column.getSqlExpression(), column.getName());
+        return format("%s AS \"%s\"", awareModel(column.getSqlExpression(), baseModel), column.getName());
+    }
+
+    private String awareModel(String expression, Model baseModel)
+    {
+        return formatSql(awareModel(parseExpression(expression), baseModel));
+    }
+
+    private Expression awareModel(Expression expression, Model baseModel)
+    {
+        return ExpressionTreeRewriter.rewriteWith(new ExpressionRewriter<>()
+        {
+            @Override
+            public Expression rewriteIdentifier(Identifier node, Void context, ExpressionTreeRewriter<Void> treeRewriter)
+            {
+                if (baseModel.getColumns().stream().anyMatch(c -> c.getName().equalsIgnoreCase(node.getValue()))) {
+                    return new DereferenceExpression(new Identifier(baseModel.getName(), true), new Identifier(node.getValue(), true));
+                }
+                return node;
+            }
+        }, expression);
     }
```

Comment:
If they have same pattern to format. how about extract the model string like below:
```java
  String model;
  if (!relationshipInfos.isEmpty() && relationableBase.isPresent()) {
      model = RelationshipRewriter.relationshipAware(relationshipInfos, relationableBase.get(), expression).toString();
  }
  else if (isMeasure) {
      model = awareModel(column.getExpression().orElseThrow(() -> new IllegalArgumentException("measure column must have expression")), baseModel);
  }
  else {
      model = awareModel(column.getSqlExpression(), baseModel);
  }
  return format("%s AS \"%s\"", model, column.getName());
```

Reply:
The return result of awareModel is a written expression. Like `orderkey` would be `Orders.orderkey`. I think that the current way is better.